frontend/awx/administration/instances/hooks/useAssociatePeersToInstance.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useAwxBulkActionDialog } from '../../../common/useAwxBulkActionDialog';
+import { requestPatch } from '../../../../common/crud/Data';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { useParams } from 'react-router-dom';
+import { useGetItem } from '../../../../common/crud/useGet';
+import { Peer } from '../../../interfaces/Instance';
+
+export function useAssociatePeersToInstance() {
+  const { t } = useTranslation();
+  const userProgressDialog = useAwxBulkActionDialog<Peer>();
+  const params = useParams<{ id?: string }>();
+  const id = Number(params.id);
+  const { data: instance } = useGetItem<Peer>(awxAPI`/instances/`, params.id);
+  const px = instance?.peers as number[];
```

Comment:
```suggestion
  const { peers }: { peers: number[]}  = instance;
```

Reply:
You can try
```
const { peers = [] } = instance ?? {};
```

Reply:
now it's this `Unsafe array destructuring of a tuple element with an `any` value`